### DISCUSSION TITLE
How to hide axis selector from the user?

### DISCUSSION THREAD

**Discussion by jleviaguirre** [TIBCOSoftware/spotfire-mods#113] [Q&A] [answered]
Is it possible to hide the axis selector from the user? I tried this on the manifest for my dataViewDefinition.axes:
```            
{
   "name": "Identifier",
   "mode": "categorical",
   "placement": "none",
   "legendItem": {"defaultVisibility": "hidden","title": ""},
   "propertyControl": {"title": "","visibility": "hidden"}
},
```

![image](https://user-images.githubusercontent.com/1098689/234918534-d4bd6e87-347e-43e3-85de-5bda4174e5b4.png)

**Comment by objerke** [comment, marked as answer]
Hi @jleviaguirre,
By setting the `"legendItem": {"defaultVisibility": "hidden"}` item the axis will not be shown when *creating a new instance* of the mod. This hides the axis from the user but it will still be accessible via the properties dialog. There is currently no way to completely hide an axis. If that is what you want I suggest you add an idea in the Idea portal.
